### DISCUSSION TITLE
How to show cluster information in Home Assistant?q

### DISCUSSION THREAD

**Discussion by corgan2222** [mKeRix/room-assistant#618] [General]
Is there a way to show how many cluster nodes are connected and which is the leader in Home Assistant?
I'm running the hass.io Plugin in HA on a RPI4 and several nodes on RP zeros. 
tnxs

**Comment by mKeRix** [comment]
Each instance will create entities called "Cluster Leader" and "Cluster Size", which will show you the information that you want. You can view those in Home Assistant under the MQTT integration (or the all entities view).

**Comment by corgan2222** [comment]
Thanks for the info. 
Maybe it's a good idea to mention this in the wiki. Didn't found it there, even after hours of clicking through each page. 
Also, it would be great for new users to make one page with a complete config example with all possible entries. ATM it a clicking orgy to go through all pages and find all needed things.

**Reply by mKeRix** [reply]
Thanks for the input - I'll bookmark this thread as a reminder to myself for improving the docs. Also feel free to edit the docs yourself if you want, it's all just Markdown files in this repo (and edit links are on each page). :)
